netdata-installer.sh

```diff
@@ -1,20 +1,24 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 
 # SPDX-License-Identifier: GPL-3.0-or-later
-# shellcheck disable=SC2046,SC2086,SC2166
+# shellcheck disable=SC2030,SC2031,SC2046,SC2086,SC2166,SC2269
```

Comment:
If we’re going through and updating things to be POSIX compliant, it would be nice to also get rid of global shellcheck overrides at the same time (that is, either fix the issues if applicable, or switch to per-line overrides, which will make it easier to ensure we’re only disabling checks when we actually need to).